src/content/en/2022/seo.md

```diff
@@ -1,22 +1,754 @@
+## Crawlability and indexability
+
+Crawling and Indexing are the backbone of what Google and other search engines ultimately display on their search results pages. Without them, ranking simply cannot happen.
+
+The first step in the process is discovering web pages via crawling. While numerous pages are crawled, fewer of them are actually indexed, which is essentially stored and categorized in a search engine’s database. Based on a searcher’s query, matching indexed pages are then served.
+
+This section deals with the state of the web, as it pertains to bots crawling and indexing websites. What directives are sites giving search engines bots? What are sites doing to ensure Google serves the right page and not a near duplicate in search results?
+
+Let’s explore the web and some of its facets that impact crawlability and indexability.
+
+
+### Robots.txt
+
+The robots.txt file instructs bots, including search engine crawlers, as to where they can and cannot go, meaning what they can or cannot crawl.
+
+{{ figure_markup(
+  image="robots-txt-status-codes.png",
+  caption="Percentages for robots.txt status codes.",
+  description="Bar chart showing percent of pages with a valid robots.txt file. 82.4% of mobile sites returned a 200 status code, and 15.8% returned a 404. A small percentage of 0.2% returned 403 and 0.1% returned 500.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSZxId5PPpTHp4835BI-rt4HqZVrhn1UiF0xcOrVUytVs6j262ZqFrY5e1FXZibAGDY6gWnZ0uG3WZC/pubchart?oid=240883913&format=interactive",
+  sheets_gid="258446623",
+  sql_file="robots-txt-status-codes.sql"
+  )
+}}
+
+There has been a nominal increase in the percentage of sites whose robots.txt files return a 200 status code in 2022 compared to 2021. In 2022,  81.5% of robots.txt files for desktop sites returned a 200 status code while 82.4% of mobile sites returned the same. This stands in comparison to 81% and 81.9% of robots.txt files on desktop and mobile sites, respectively, returning a 200 status code in 2021.
+
+Concurrently, there was just a small reduction in the percentage of robots.txt files returning a 404 status code in 2022 compared to 2021. Last year, 17.3% of robots.txt files for desktop sites returned a 404 while 16.5% of mobile sites’ robots.txt files returned that status code. In 2022, it’s just 16.5% for desktop and 15.8% for mobile sites’ robots.txt files that are returning a 404 status code.
+
+Like in 2021, the remaining status codes are associated with a minimal number of robots.txt files.
+
+Note: The above data does not indicate how well optimized a robots.txt file is. Even a file returning a 200 status code can contain directives that are perhaps not in the best interest of a site’s overall health.
+
+### Robots.txt size
+
+{{ figure_markup(
+  image="robots-size.png",
+  caption="96% of robots.txt files are between 0 and 100 kilobytes.",
+  description="A distribution chart showing differences in robots.txt sizes in 100 kilobyte increments.",
```

Comment:
Can we review the descriptions on all the charts? Many of them are quite high level and don't fully describe the data.

Reply:
These are updated now.